### ISSUE TITLE
Add the led fx integration name before the entity

### ISSUE THREAD

**Issue by DivanX10** [dmamontov/hass-ledfx#2]
Can you add the Led FX integration name to your integration before the entity name? This is necessary to eliminate confusion about what the entity is and what it is responsible for

This is what the name of the WLED entity looks like
![image](https://user-images.githubusercontent.com/64090632/146577859-c54eb2a4-3dd7-41af-ad18-1a8b04549f8b.png)

This is what the name of the WLED entity looks like through Led FX integration. Where you can see that the number 2 is at the end, which is not very correct. It would be necessary to make the led fx_ stand in front of any entity, so that it looks like this light.ledfx_wled_monitor
![image](https://user-images.githubusercontent.com/64090632/146577818-faef8761-08f9-49df-afed-3c69c8aa4993.png)

***

Of course I can rename the devices in the LedFX addon itself, but it would probably be better if for some reason the devices were not renamed or forgot to rename them, then before the entity name should be added ledfx

![image](https://user-images.githubusercontent.com/64090632/146580151-7c14cccb-6d93-489c-b9a6-8c2509e1a2c7.png)
![image](https://user-images.githubusercontent.com/64090632/146580187-c25a6093-a3a3-4e1b-b9db-91e729bf919c.png)

**Comment by dmamontov** [comment]
the integration uses id and not name to generate the code, this is correct. I think I can do it as an extreme opportunity. But this will not be a priority. You can always just change this in your hass settings.

**Comment by zigomatichub** [comment]
Thank you for your job Dimitri.

I'm searching for the same as well. I don't know how to combine all entities (or attributes). It's difficult to find them. 

I try the example in performance documentation to have the card as you show but I didn't succeed
This part is not working for me ( device id, I tried a lot of things but never have any result) 
It seems that auto-entities have changed because this combination is not working. I tried a lot of other combinations but no luck

      include:
        - attributes:
            device: <your_device_id>
          not:
            state: unavailable

**Comment by dmamontov** [comment]
I'm a little out of the loop right now. You can send the full configuration of the card so that I can see all the attributes of the device.

**Comment by zigomatichub** [comment]
No problem Dimitri. 
I found that device_id in your documentation is refering to the Ledfx console information. None of the display in HA ledfx integration mention the same name.

I expose another situation: 
I have 2 ledfx instance on 2 servers and when I integrated both in HA, wled devices from the second ledfx instance was also present in the first ledfx instance. This was giving a little bit of mess...   So I removed one of the 2 as a workaround. 

Be safe and in peace
Zigomatic
